src/Microsoft.AspNetCore.Razor/CodeGenerators/Visitors/CSharpTagHelperFieldDeclarationVisitor.cs

```diff
@@ -168,6 +188,23 @@ private void PreAllocateUnboundTagHelperAttributes(TagHelperChunk chunk)
             }
         }
 
+        private static bool IsLiteralAttributeValue(Chunk attributeValueChunk)
```

Comment:
ok. I saw some other chunks like ExpressionChunks. Don't we need to worry about them?